Tests/BudgetSquirrel.TestUtils/Seeds/BasicSeed.cs

```diff
@@ -0,0 +1,103 @@
+// using BudgetSquirrel.Business.Auth;
```

Comment:
Is there a reason why you just commented out these files instead of delete them?